cypress/integration/BasicSearch.test.ts

```diff
@@ -33,7 +33,7 @@ describe("Basic Search", () => {
   it("generates search history", () => {
     cy.restoreLocalStorage();
     // history only shows the invalid search because cy.saveLocalStorage(); overwrites cy.saveLocalStorage(); in example search
-    cy.get('[data-testid="historyButton"]').click();
+    cy.get('[data-testid="historyButton"]').click({ force: true });
```

Comment:
This brakes the test in gh actions but it work fine locally. I will look into this tomorrow.

`{ force: true }` was used because cypress could not see b-navbar-items inside b-navbar